torch/csrc/jit/passes/dead_code_elimination.cpp

```diff
@@ -259,6 +260,14 @@ class DeadCodeEliminator {
 
   // Delete all unmarked nodes.
   void sweep(Block* block, bool recurse) {
+    auto allowed = JIT_BISECT();
+    if (!allowed) {
+      JIT_BISECT_LOG(
+        std::string("Stop deleting because of opt limitation"),
```

Comment:
Nit: generally it might be a good idea to "standardize" such messages across passes.